tools/perf.sh

```diff
@@ -85,12 +85,18 @@ if [[ -z "$DATAFILE" ]]; then
     DATAFILE=perf.data
 fi
 if [[ -z "$SPACETIME_PID" ]]; then
-    if [[ $(ps -a | grep spacetime | wc -l) != 1 ]] ; then
+    SPACETIMES="$(ps -a -e -f | grep '\<spacetime\>.*\<start\>' | grep -v '\<grep\>')"
```

Comment:
`-f` allows us to get the args
`-e` lets us see all users

Reply:
This would make a great comment to add to the code.

Reply:
Although, that's a bigger change